Classes/ZedBaseSiren.uc

```diff
@@ -150,10 +160,35 @@ function HurtRadius(float DamageAmount, float DamageRadius, class<DamageType> Da
                     HitLocation);
         }
     }
+
     bHurtEntry = false;
 }
 
 
+// DEAD sirens can NOT spawn scream emitter 
```

Comment:
Actually, then can. The idea is to allow Siren to continue screaming after death unless **decapitated**. There is a `bDecapitated` check inside `SpawnTwoShots()` to prevent dealing damage if decapitated. I think we can move further by stopping the animation on decapitation. But I want to keep Siren's death scream if killed by body shots.

```
if (bDecapitated)
    StopAnimating();
```

Reply:
Well, your choice. It just looks weird when you kill the siren, she drops down and then siren emitter spawns where her head was. TWI's "animation does all the job" aproach feels bad.

Reply:
On second thought, I think you are right. I'd like dead but not headless Sirens to keep screaming, but network sync kills immersion. For instance, badly thrown nade could push Siren towards players and damage them while flying. However, ragdoll physics applies client-side only, while server-side, the Siren stays where she died. OK, let's leave your code as-is - stop screaming once dead no matter of decapitaion.